src/rollup/rollup.ts

```diff
@@ -29,9 +29,9 @@ import { error } from "../error.ts";
  * @public
  */
 export async function rollup(
-  options: RollupOptions,
+  options: InputOptions,
```

Comment:
We’re gonna want to leave this as `RollupOptions` so that folks can pass their Rollup config in and not receive type errors ( as it includes the `output` object ). Should be a case of providing our own version sim to  `InputOptions` in the type exports which extends our version of the input options over the core Rollup module type.

REF: https://github.com/rollup/rollup/blob/master/src/rollup/types.d.ts#L785